handlers/main.yml

```diff
@@ -1 +1,6 @@
 ---
+
+- name: php_fpm restart php-fpm
+  service:
+    name: rh-php71-php-fpm.service
```

Comment:
change to `rh-php72-php-fpm.service`. I suggest putting the php version in a variable

Reply:
I'd suggest to also remove the `.service`.